packages/gatsby/src/commands/build.js

```diff
@@ -150,16 +150,20 @@ module.exports = async function build(program: BuildArgs) {
     })
   } catch (err) {
     let id = `95313` // TODO: verify error IDs exist
-    if (err.message === `ReferenceError: window is not defined`) {
+    const context = {
+      errorPath: err.context && err.context.path,
+    }
+
+    const match = err.message.match(/ReferenceError: (\w+) is not defined/)
```

Comment:
I'm bit hesitant on using `\w+` here (as it will catch any variables and not just browser-only ones - should we have list of potential variables (`(browser|navigator|localStorage)` etc) here instead maybe?

Reply:
@pieh good idea! I will fix it.

Reply:
@pieh I suggest to add only a most common list `window|document|localStorage|navigator|alert|location`, because browser global scope contains a lot more variables.

Reply:
Sounds perfect! We can always add more browser-only global variables here in future